src/Components/Patient/ManagePatients.tsx

```diff
@@ -577,6 +577,16 @@ export const PatientManager = () => {
                       text="Readmission"
                     />
                   )}
+                  {patient.last_consultation?.suggestion === "A" &&
+                    patient.last_consultation.facility === patient.facility &&
```

Comment:
what is this check for? seems redundant!

Reply:
It is to check whether the last_consultation is same as patient's active facility or not.

Since patient could be transferred to another facility without a consultation filed yet. So in UI it'd be shown as "No Consultation Filed" but shows IP days as per last consultation which is already closed consultation.

Ref: https://github.com/coronasafe/care_fe/pull/7252#issuecomment-2082313077